src/act.other.cpp

```diff
@@ -4637,22 +4637,22 @@ ACMD(do_syspoints) {
     }
 
     // Mail the victim.
-    snprintf(buf, sizeof(buf), "You have been %s %d system points for %s.\r\n",
+    snprintf(buf, sizeof(buf), "You have been %s %d system points for%s\r\n",
             (award_mode ? "awarded" : "penalized"),
             k,
             reason);
     store_mail(idnum, ch, buf);
 
     // Notify the actor.
-    send_to_char(ch, "You %s %d system points %s %s for %s.\r\n",
+    send_to_char(ch, "You %s %d system points %s %s for %s\r\n",
                 (award_mode ? "awarded" : "penalized"),
                 k,
                 (award_mode ? "to" : "from"),
                 capitalize(target),
                 reason);
 
     // Log it.
-    snprintf(buf, sizeof(buf), "%s %s %d system points %s %s for %s (%d to %d).",
+    snprintf(buf, sizeof(buf), "%s %s %d system points %s %s for %s (%d to %d)",
```

Comment:
This is a log message that the player doesn't see, and the staff member cannot alter the ending punctuation here. Please restore the period.

Reply:
Will lead to messages of "staff awarded x syspoints to player for thus and such !!!!!."

Thought I was helping future readability.